### PR TITLE
Add message template support

### PR DESCRIPTION
**Pull request by r3nic1e**
This PR implements #3. For backward compatibility old formatting style is used when no template is defined.

It uses code from alertmanager, so configuration is similar to alertmanager's one:
```yaml
providers:

templates:
- /template.yml

receivers:
- name: test
  provider: telegram
  text: '{{ template "telegram_message" . }}'
```

### CODE REVIEW

**Comment by epels** [comment]
Hi @r3nic1e, thanks for opening a pull request! It's great that you've been able to implement this in a backwards compatible manner. We'll be looking at it shortly.

File: cmd/sachet/config.go [hunk 1]
```diff
@@ -57,5 +61,10 @@ func LoadConfig(filename string) error {
 		return err
 	}
 
+	tmpl, err = template.FromGlobs(config.Templates...)
+	if err != nil {
+		return err
```
Comment (epels) [inline cmd/sachet/config.go]: This can be simplified to simply `return err`

File: cmd/sachet/config.go [hunk 2]
```diff
@@ -24,6 +25,7 @@ type ReceiverConf struct {
 	Provider string
 	To       []string
 	From     string
+	Text     string
```
Comment (epels) [inline cmd/sachet/config.go]: What do you think about renaming this to `Template`?

Comment (epels) [reply cmd/sachet/config.go]: @marcelcorso Agreed, thanks for clarifying!

Comment (r3nic1e) [reply cmd/sachet/config.go]: I made everything to be similar to alertmanager configuration and thought about some  providers that support message titles or something like that.

I'd prefer current name but don't mind renaming.

Comment (epels) [reply cmd/sachet/config.go]: @r3nic1e No, this is perfect - I wasn't aware that's what Prometheus calls it, so let's be consistent then 👍

**Comment by epels** [comment]
Released as version [0.2.0](https://github.com/messagebird/sachet/releases/tag/0.2.0)!

**Pull request merged**